src/pool/srv_pool.c

```diff
@@ -3640,7 +3640,10 @@ ds_pool_check_leader(uuid_t pool_uuid, daos_unit_oid_t *oid,
 	if (rc != 1)
 		D_GOTO(out, rc = -DER_INVAL);
 
-	crt_group_rank(pool->sp_group, &myrank);
+	rc = crt_group_rank(NULL, &myrank);
```

Comment:
Looks we still use subgroup rank in other places, should we cleanup them all together in same patch? Liwei, any thoughts? I'm not sure if you have already started the cleanup work on this.

Reply:
Some IV API methods seem to be expecting secondary group ranks. I haven't started looking through all crt_group_rank calls yet; would like to land my existing bootstrapping patches first to avoid conflicts.